src/common/transformations/src/transformations/common_optimizations/matmul_split_decomposition.cpp

```diff
@@ -0,0 +1,209 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/common_optimizations/matmul_split_decomposition.hpp"
+
+#include "itt.hpp"
+#include "openvino/core/rt_info.hpp"
+#include "openvino/opsets/opset1.hpp"
```

Comment:
it's not recommended to use `#include "openvino/opsets/opset1.hpp"` opsetN headers in the transformation code due to it leads to significant increase in ov lib sizes in debug build.
it's better to use the concrete op versions, e.g. ov::op::v1::